graphql2/graphqlapp/escalationpolicy.go

```diff
@@ -55,6 +57,14 @@ func (m *Mutation) CreateEscalationPolicyStep(ctx context.Context, input graphql
 		)
 	}
 
+	if len(input.Targets) != 0 {
```

Comment:
This isn't necessary as `for ... range` already handles nil/empty slice correctly

Reply:
Got it, just removed it